ospf_suppress_fa -> test failure every time for me.

```================================================================================ short test summary info ================================================================================
FAILED test_ospf_suppress_fa.py::test_converge_protocols - AssertionError: r1 has not converged
FAILED test_ospf_suppress_fa.py::test_ospf_set_suppress_fa - AssertionError: Unable to get expected initial states
============================================================================= 2 failed in 65.66s (0:01:05) ===================
```


Git bisect leads me to this commit:

```
commit 390081329874c4fd8b4a15ffa2a090275b5fc8ac (HEAD, refs/bisect/bad)
Author: Igor Ryzhov <iryzhov@nfware.com>
Date:   Wed Jan 24 23:04:23 2024 +0200

    zebra: convert to mgmtd
    
    Signed-off-by: Igor Ryzhov <iryzhov@nfware.com>
```

--- a/tests/topotests/ospf_suppress_fa/r2/ospfd.conf
+++ b/tests/topotests/ospf_suppress_fa/r2/ospfd.conf
@@ -10,6 +10,7 @@ interface r2-eth1
  ip ospf dead-interval 10
 !
 router ospf
+ ospf router-id 10.0.23.2
  network 10.0.12.0/24 area 0
  network 10.0.23.0/24 area 1
  area 1 nssa 

